src/index.html

```diff
@@ -1,11 +1,18 @@
 <!doctype html>
 <html>
-  <head>
-    <script src="https://cdn.jsdelivr.net/npm/@tensorflow/tfjs"></script>
-    <script src="https://cdn.jsdelivr.net/npm/@tensorflow-models/speech-commands"></script>
-  </head>
-  <body>
-    <div id="console"></div>
-    <script src="index.js"></script>
-  </body>
-</html>
\ No newline at end of file
+
+<head>
+  <script src="https://cdn.jsdelivr.net/npm/@tensorflow/tfjs"></script>
+  <script src="https://cdn.jsdelivr.net/npm/@tensorflow-models/speech-commands"></script>
+</head>
+
+<body>
+  <button id="left" onmousedown="collect(0)" onmouseup="collect(null)">Left</button>
```

Comment:
Es wäre schöner, wenn du die Funktionalität in einer Angular-Komponente entwickelst und nicht im index.html. Die Scripts können hier geladen werden, das ist ok.